cmd/soroban-cli/src/commands/config/mod.rs

```diff
@@ -63,15 +63,28 @@ pub struct Args {
     pub ledger_file: ledger_file::Args,
 
     #[arg(long, alias = "source", env = "SOROBAN_ACCOUNT")]
-    /// Account that signs the final transaction.
-    /// S...          a seceret key
-    /// alice         an identity
-    /// 'kite urban.  a seed phrase
-    /// DEFAULT       Is the key generated with `identity generate --seed 0000000000000000
+    /// Account that signs the final transaction. Alias `source`
+    ///
+    /// SC36BWNUOCZA...  a secret key
+    ///
+    /// alice            an identity
+    ///
+    /// "kite urban .."  a seed phrase
+    ///
+    /// If not provided the default is the seed phrase generated by  
+    /// `identity generate --default-seed`
+    ///
+    /// Examples:
+    ///
+    ///     --source alice
+    ///
+    ///     --source "kite urban.."
+    ///
+    ///     --source SC36BWNUOCZAO7DMEJNNKFV6BOTPJP7IG5PSHLUOLT6DZFRU3D3XGIXW
```

Comment:
Since multiline argument comments are not well supported in any context, how about this:

```suggestion
    /// Account that signs the final transaction. Alias `source`. Can be an identity (--source alice), a secret key (--source SC36…), or a seed phrase (--source "kite urban…"). Default: `identity generate --default-seed`
```